packages/fast-components-styles-msft/src/utilities/color/palette.ts

```diff
@@ -0,0 +1,167 @@
+import { neutralPaletteSource } from "./color-constants";
+import {
+    DesignSystem,
+    DesignSystemResolver,
+    ensureDesignSystemDefaults,
+} from "../../design-system";
+import chroma from "chroma-js";
+import { memoize } from "lodash-es";
+import { colorMatches, isValidColor, luminance } from "./common";
+import { neutralForegroundDark, neutralForegroundLight } from "./neutral-foreground";
+
+/**
+ * The named palettes of the MSFT design system
+ */
+export enum Palettes {
+    neutral = "neutral",
+    accent = "accent",
+}
```

Comment:
nit: prefer the name of the enum to be DefaultPalette or something because we usually don't have enums as plurals, eg. an enum with a set of possible colors would be Color.red, Color.blue